frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -2073,7 +2073,10 @@ export class Arena {
           language: language,
         };
         this.updateRun(run);
-        if (this.runSubmitView) {
+        if (
+          this.runSubmitView &&
+          Object.getOwnPropertyNames(this.runSubmitView?.$refs).length !== 0
```

Comment:
```suggestion
          Object.getOwnPropertyNames(this.runSubmitView.$refs).length !== 0
```

esto falla si se pasa `undefined`, así que el nullish coalescing operator en realidad no está ayudando. también, en la línea anterior ya se aseveró que no es nullish, así que se puede omitir sin problema.

Reply:
Listo.